<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDataWindow</class>
 <widget class="QMainWindow" name="GetDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lStart">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lEnd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>End:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teEnd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lStep">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teStep">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbType">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>JSON</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GRAPH</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFetch">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>130</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCF">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CF:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCF">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Average</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Max</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Last</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lFile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>98</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teFile">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
